core/modules/replica/testConfiguration.cc

```diff
@@ -268,77 +267,77 @@ BOOST_AUTO_TEST_CASE(ConfigurationTest) {
 
         // Databases
 
-        std::vector<std::string> databases1 = config->databases();
-        std::sort(databases1.begin(), databases1.end());
+        vector<string> databases1 = config->databases();
+        sort(databases1.begin(), databases1.end());
         BOOST_CHECK(databases1.size() == 5);
-        BOOST_CHECK(databases1 == std::vector<std::string>({"db1","db2","db3","db4","db5"}));
+        BOOST_CHECK(databases1 == vector<string>({"db1","db2","db3","db4","db5"}));
 
-        std::vector<std::string> databases2 = config->databases("production");
-        std::sort(databases2.begin(), databases2.end());
+        vector<string> databases2 = config->databases("production");
+        sort(databases2.begin(), databases2.end());
         BOOST_CHECK(databases2.size() == 3);
-        BOOST_CHECK(databases2 == std::vector<std::string>({"db1","db2","db3"}));
+        BOOST_CHECK(databases2 == vector<string>({"db1","db2","db3"}));
 
-        std::vector<std::string> databases3 = config->databases("test");
-        std::sort(databases3.begin(), databases3.end());
+        vector<string> databases3 = config->databases("test");
+        sort(databases3.begin(), databases3.end());
         BOOST_CHECK(databases3.size() == 2);
-        BOOST_CHECK(databases3 == std::vector<std::string>({"db4","db5"}));
+        BOOST_CHECK(databases3 == vector<string>({"db4","db5"}));
     
-        for (auto&& name: std::vector<std::string>({"db1","db2","db3","db4","db5"})) {
+        for (auto&& name: vector<string>({"db1","db2","db3","db4","db5"})) {
```

Comment:
I'd prefer spaces after commas, I don't know if we have a coding standard rule about it tho.

Reply:
(this is also a note to myself for the future reviews) I think the review should be based on adopted conventions (a way most of the code is written in package `qserv`), common standards (including LSST Style Guide, etc.), and manager's requirements... rather than on personal preferences of reviewers (unless the reviewer is the manager) 😸

Seriously speaking, for the former (the standards) I've found such requirement in the Style Guide:
```
Commas should be followed by a white space.
```
Details in:
https://developer.lsst.io/cpp/style.html#white-space
So, I will fix the code accordingly.